habitat-lab/habitat/tasks/rearrange/utils.py

```diff
@@ -736,13 +751,21 @@ def get_camera_transform(cur_articulated_agent) -> mn.Matrix4:
     else:
         raise NotImplementedError("This robot does not have GazeGraspAction.")
 
-    # Get the camera's attached link
-    link_trans = cur_articulated_agent.sim_obj.get_link_scene_node(
-        cam_info.attached_link_id
-    ).transformation
-    # Get the camera offset transformation
-    offset_trans = mn.Matrix4.translation(cam_info.cam_offset_pos)
-    cam_trans = link_trans @ offset_trans @ cam_info.relative_transform
+    cam_trans = None
+
+    if deprecated_logic:
```

Comment:
Interesting strategy. Do we need this? Are there active down-stream use cases we are tracking? 

IMO We can change the logic without back-compat and mark this PR as a breaking change since it is so close to the last release.

Reply:
sounds good to me! priyam you made a call for this!

Reply:
Thanks! I'll take this out so we have a cleaner code-merge.